apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -15,17 +15,17 @@
 
   <!-- configuration options exposed to javascript -->
   <meta id="ood_config"
-    data-bg-color="<%= Configuration.brand_bg_color %>"
-    data-link-bg-color="<%= Configuration.brand_link_active_bg_color %>"
+    data-bg-color="<%= @user_configuration.brand_bg_color %>"
+    data-link-bg-color="<%= @user_configuration.brand_link_active_bg_color %>"
   />
 </head>
 <body>
   <header>
-    <nav class="navbar navbar-expand-md shadow-sm navbar-color navbar-<%= Configuration.navbar_type %>" aria-label="Navigation">
-    <% if ENV['OOD_DASHBOARD_HEADER_IMG_LOGO'].present? %>
-      <a class="navbar-brand navbar-brand-logo" href="<%= root_path %>"><img class="img-fluid" src="<%= ENV['OOD_DASHBOARD_HEADER_IMG_LOGO'] %>" alt="<%= OodAppkit.dashboard.title.html_safe %>"></a>
+    <nav class="navbar navbar-expand-md shadow-sm navbar-color navbar-<%= @user_configuration.navbar_type %>" aria-label="Navigation">
+    <% if @user_configuration.dashboard_header_img_logo %>
+      <a class="navbar-brand navbar-brand-logo" href="<%= root_path %>"><img class="img-fluid" src="<%= @user_configuration.dashboard_header_img_logo %>" alt="<%= @user_configuration.dashboard_title.html_safe %>"></a>
```

Comment:
```suggestion
      <a class="navbar-brand navbar-brand-logo" href="<%= root_path %>"><img class="img-fluid" src="<%= @user_configuration.dashboard_header_img_logo %>" alt="<%= @user_configuration.dashboard_title %>"></a>
```

`html_safe` is not what we think. I know you didn't write this, but we may as well fix it when I see it.